pkg/gatherers/workloads/workloads_info.go

```diff
@@ -174,24 +191,56 @@ func gatherWorkloadInfo(
 		info.Namespaces[namespaceHash] = namespacePods
 		info.PodCount += namespacePods.Count
 	}
+	return limitReached, info, imageCh, nil
+}
 
-	workloadImageResize(info.PodCount)
+func podIsTerminal(pod *corev1.Pod) bool {
+	return pod.Status.Phase == corev1.PodSucceeded ||
+		pod.Status.Phase == corev1.PodFailed
+}
 
-	records := []record.Record{
-		{
-			Name: "config/workload_info",
-			Item: record.JSONMarshaller{Object: &info},
-		},
+func podCanBeIgnored(pod *corev1.Pod) bool {
```

Comment:
Some similar name suggestions as above and this would definitely deserve some explaining comment, because I don't know when a pod can be ignored.

Reply:
This conversation doesn't seem to be resolved.